.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+	"python.pythonPath": "venv/bin/python",
+	"python.formatting.provider": "black"
+}
```

Comment:
Please pull .vscode/ from the commit. This is my mistake, it should be in the .gitignore file.